### PR TITLE
snowflakes, they melt eventually.

### PR DESCRIPTION
**Pull request by SgtxRootbeer**
:cl: a stressed, annoyed, and angered hits
tweak:tweak
add: snowflake lockers and accompanying room
tweak: moved ai upload stuff to ai chamber, more turrets added to compensate.
add: new room, to act as a stop between the locker room, and the ai core. it contains the turret controller.
fix: cmo's door
fix: ssu underneath the blackbox
/:cl:

### CODE REVIEW

**Comment by SgtxRootbeer** [comment]
also, wooo, pr#100

File: _maps/map_files/Trailblazer/Trailblazer.dmm [hunk 1]
```diff
@@ -12714,6 +12760,16 @@
 	dir = 1
 	},
 /area/shuttle/ftl/security/armory)
+"BiS" = (
+/obj/item/clothing/under/shorts/black,
+/obj/item/clothing/head/bearpelt,
+/obj/structure/closet/secure_closet/personal{
+	desc = "I'm gay";
```
Comment (francinum) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: Nice.

Comment (SgtxRootbeer) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: capture
this is his application.

Comment (francinum) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: Yes, I saw it. I was there for this. Who else do you think checks this fork?

Comment (Pascal125) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: Kek.

**Comment by SgtxRootbeer** [comment]
i feel dirty for having mapped in these last 2 commits, due to the sheer number of guns and whatnot added to them.

**Pull request merged**